lib/puppeteer-helpers.js

```diff
@@ -1,5 +1,29 @@
 const { ports } = require('../config')
 
+/**
+ * Smooth, slower scroll to bottom of page
+ *
+ * @param {import('puppeteer').Page} page
+ */
+async function autoScroll (page) {
```

Comment:
Using something like `scrollIntoView` doesn't work here, even with `behaviour: smooth` on - it seems like it's too quick for `iframe-resizer` to catch.

Hence this slower scroll, where we scroll the `window.innerHeight` in increments till we get to the bottom of the page, ensuring all iFrames are loaded.